yarn-examples/yarn-v2/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "yarn-v2",
```

Let's also rename the name of the directory
```suggestion
  "name": "yarn-berry",
```